tables_app/src/androidTestUitest/java/org/opendatakit/espresso/TablePrefTest.java

```diff
@@ -545,21 +540,28 @@ public void display_outOfAppDirViewFile() {
   }
 
   @Test
-  public void display_badFormId() throws ServicesAvailabilityException {
+  public void display_badFormId() {
```

Comment:
After reviewing this more carefully I am not sure we need the catch statement at all.

Therefore, you can do the assert with the activity -> { (assert here) }

And eliminate the need for a weird final array.